tests/core/test_open_in_process.py

```diff
@@ -345,4 +367,4 @@ async def do_sleep_forever():
     # exception along when returning control to open_in_proc(), which will interpret it as a
     # failure of the process to exit and send a SIGKILL (together with a warning).
     if above_limit_proc_created:
-        assert False, "This process must not have been created successfully"
+        assert False, "This process must not have been created successfully"  # noqa: B011
```

Comment:
You can do this as `raise AssertionError(...)` if you want to avoid the `# noqa`